packages/node/src/bugsnag.ts

```diff
@@ -6,8 +6,6 @@ import assign from '@bugsnag/core/lib/es-utils/assign'
 import { schema as baseConfig } from '@bugsnag/core/config'
 import browserConfig from './config'
 
-import Event from '@bugsnag/core/event'
```

Comment:
Using this export from `'@bugsnag/core/event'` was causing `'@bugsnag/core/event'` to appear in the generated types, which was causing issues with some older versions of TypeScript because it would need to resort to `package.exports`, which requires a certain version of TypeScript and `moduleResolution: bundler`.

Reply:
I get the feeling a lot of these have come from auto-complete

Reply:
Yeah, it would be good to get around preventing invalid imports with eslint rules as we do on the dashboard